print/cups/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        OpenPrinting cups 2.4.2 v
+github.tarball_from releases
+distname            ${github.project}-${github.version}-source
+revision            0
+categories          print
+license             Apache-2
+maintainers         nomaintainer
+description         CUPS is the current standards-based, open source printing system developed by \
+                    OpenPrinting for Unix-like operating systems and uses IPP Everywhere™ to support \
+                    printing to local and network printers.
```

Comment:
This short description is much longer than most ports' short descriptions which rarely are longer than one line.